projects/Field Service - Manager/Controllers/fs_asset.Assets-Location.listxml

```diff
@@ -34,7 +34,7 @@
           <CellKind>0</CellKind>
           <Anchor>13</Anchor>
         </CellTemplate>
-        <CellTemplate SchemaName="Home.WindMill.png">
+        <CellTemplate SchemaName="Home. WindMill.png">
```

Comment:
please remove space at beginning of image name

Reply:
space removed at the beginning of the image name.